Realm-JSONAPI/Classes/RLMObject+JSONAPISerializer.m

```diff
@@ -29,11 +29,7 @@ - (NSMutableDictionary *)toJSONWithIncludedTypes:(NSSet *)types
   NSDictionary        *map = [[self class] JSONtoModelMap];
   RLMSchema           *schema = self.realm ? self.realm.schema : [RLMRealm defaultRealm].schema;
   Class                class = [self class];
-  // Realm often subclasses our objects with RLMAccessor, or RLMStandalone if we're not in a Realm
-  if ([NSStringFromClass(class) rangeOfString:@"RLMAccessor"].location != NSNotFound
-      || [NSStringFromClass(class) rangeOfString:@"RLMStandalone"].location != NSNotFound) {
-    class = [class superclass];
-  }
```

Comment:
any reason that these code blocks were removed? did Realm 2.10 fix the need for this?

Reply:
In testing with Realm 2.10.1 `[self class]` was of type RLM:Managed (previously RLM:Accessor) which resulted in never grabbing the correct superclass